singer/catalog.py

```diff
@@ -109,3 +110,29 @@ def get_stream(self, tap_stream_id):
             if stream.tap_stream_id == tap_stream_id:
                 return stream
         return None
+
+
+CATALOG_SCHEMA = {'type': 'object',
+                  'required': ['streams'],
+                  'properties': {
```

Comment:
I think you might want to add `"additionalProperties": false` here and in the other nested object in this schema. If you don't the validator will allow objects with other properties. I guess it depends on how strict we want to be in this validation. @cmerrick what are your thoughts on that?